cypress/locators/loader/locators.js

```diff
@@ -0,0 +1,3 @@
+// component preview locators
+export const LOADER = '[data-component="loader"]';
+export const LOADER_INSIDE_BUTTON = '[data-component="button"]';
```

Comment:
I think we already have defined such locator in common or in button `locators.js` file

Reply:
fixed